XenAdmin/Controls/CheckableDataGridView/CheckableDataGridViewRow.cs

```diff
@@ -78,9 +78,13 @@ public string DisabledReason
 
         public bool LockDisabledState { get; set; }
 
-        public abstract bool WarningRequired { get; }
+        public abstract bool LicenseWarningRequired { get; }

```

Comment:
I tried to avoid changing the structure as much as possible (though I had the impulse to rewrite it from scratch a couple of times)
Out of scope for this PR (which has significantly increased in size already)